src/docs/custom-layers/1-combination-layer.md

```diff
@@ -0,0 +1,152 @@
+<!-- INJECT:"CombinationLayer" heading -->
```

Comment:
consider dropping the heading. Heading adds that behavior on mouseover that extends the chart to 70% of screen. in our original tutorial I found that distracting especially when interacting with table of contents. eventually, do what you think works best